packages/openactive-integration-tests/test/templates/b-req.js

```diff
@@ -489,45 +490,30 @@ function createStandardBWithoutAcceptedOffer(data) {
   return null;
 }
 
-/**
- * B request with additional details required, but not supplied
- *
- * @param {BReqTemplateData} data
- */
-function createAdditionalDetailsRequiredNotSuppliedBReq(data) {
-  const req = createStandardPaidBReq(data);
-  return additionalDetailsRequiredNotSupplied(req);
-}
-
 /**
  * B request with additional details required and supplied
  *
  * @param {BReqTemplateData} data
  */
 function createAdditionalDetailsRequiredAndSuppliedBReq(data) {
```

Comment:
[OPTIONAL - HAPPY FOR IGNORE DUE TO TIME PRESSURE] Maybe lose the "required" part of this. This is a hangover from the previous work where the request somehow specified that additional details were required. But now you've fixed it so that the request only supplies additional details and has no say on whether or not they are required (e.g. you could use this for a "test what happens when additional details are not required but they are supplied anyway" kind of test)

Reply:
Yeh very fair

Reply:
@civsiv you've marked this as resolved but it has not changed. Is that a mistake or have you decided to not do it for now?

Reply:
Ah sorry, removed